.github/ISSUE_TEMPLATE/user_bug_report.yml

```
@@ -0,0 +1,89 @@
+name: User bug report
+description: Create a report to help us improve
+title: '[User reported bug]: '
+labels: C-bug, S-needs-triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "I expected to see this happen:
+
+
+      Instead, this happened:
+
+    "
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Copy and paste the exact commands or code here.
+      placeholder: "Behavior or code sample that causes the bug"
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Zebra logs
+      description: Copy and paste the last 100 Zebra log lines or upload the full logs to https://gist.github.com/ and add a link to them here.
+      placeholder: "copy and paste the logs here"
+    validations:
+      required: false
+  - type: checkboxes
+    id: crate
+    attributes:
+      label: In what crates the bug is located, if in any?
+      description: You may select more than one.
+      options:
+        - label: zebrad
+        - label: zebra-chain
+        - label: zebra-consensus
+        - label: zebra-network
+        - label: zebra-node-services
+        - label: zebra-rpc
+        - label: zebra-script
+        - label: zebra-state
+        - label: zebra-test
+        - label: zebra-utils
+        - label: tower-batch
+        - label: tower-fallback
+  - type: input
+    id: zebrad-version
+    attributes:
+      label: Zebra Version
+      description: "For bugs in `zebrad`, run `zebrad version`."
```

This command got deleted yesterday in PR #6801
```suggestion
      description: "For bugs in `zebrad`, run `zebrad --version`."
```